modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceProductMedia.testcase

```diff
@@ -320,6 +320,226 @@ definition {
 		}
 	}
 
+	@description = "COMMERCE-12773 | Verify that it is possible to use the toggle Shopping Experience Image Gallery to decide which image shows on the storefront product details"
+	@priority = 4
+	test CanUseTheShoppingExperienceImageGalleryToggle {
```

Comment:
How about this name - CanViewImagesCorrectlyByUsingTheShoppingExperienceImageGalleryToggle